src/Microsoft.Restier.EntityFramework/DbApi.cs

```diff
@@ -56,71 +57,40 @@ protected T DbContext
         /// </returns>
         protected override ApiBuilder ConfigureApi(ApiBuilder builder)
         {
-            builder = base.ConfigureApi(builder);
-            builder.CutoffPrevious<IModelBuilder>(ModelProducer.Instance);
-            builder.CutoffPrevious<IModelMapper>(new ModelMapper(typeof(T)));
-            builder.CutoffPrevious<IQueryExpressionSourcer, QueryExpressionSourcer>();
-            builder.CutoffPrevious<IQueryExecutor>(QueryExecutor.Instance);
-            builder.CutoffPrevious<IChangeSetPreparer, ChangeSetPreparer>();
-            builder.CutoffPrevious<ISubmitExecutor>(SubmitExecutor.Instance);
-            return builder;
-        }
-
-        /// <summary>
-        /// Creates the API context for this API.
-        /// </summary>
-        /// <param name="configuration">
-        /// The API configuration to use.
-        /// </param>
-        /// <returns>
-        /// The API context for this API.
-        /// </returns>
-        protected override ApiContext CreateApiContext(
-            ApiConfiguration configuration)
-        {
-            var context = base.CreateApiContext(configuration);
-            var dbContext = this.CreateDbContext();
+            builder = base.ConfigureApi(builder)
+                .CutoffPrevious<IModelBuilder>(ModelProducer.Instance)
+                .CutoffPrevious<IModelMapper>(new ModelMapper(typeof(T)))
+                .CutoffPrevious<IQueryExpressionSourcer, QueryExpressionSourcer>()
+                .CutoffPrevious<IQueryExecutor>(QueryExecutor.Instance)
+                .CutoffPrevious<IChangeSetPreparer, ChangeSetPreparer>()
+                .CutoffPrevious<ISubmitExecutor>(SubmitExecutor.Instance);
+            builder.Services
+                .AddScoped<T>(sp =>
```

Comment:
The same question here: Shall we just use `GetService<DbContext>` so here you don't have to register the service for `T` (the actual type of `DbContext`)?